public/media_engagement/ahadi_media_engagement.html

```diff
@@ -6,9 +6,14 @@
     <link rel="shortcut icon" type="image/png" href="../../public/coda/favicon.png" />
     <title>Engagement & Media Dashboard</title>
     <!-- Compiled and minified CSS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```

Comment:
Where is this used?

Reply:
The navbar components require that to function. It's required to add that and bootstrap js in order for them to work.

Reply:
Popper.js is a positioning engine; its purpose is to calculate the position of an element to make it possible to position it near a given reference element

Reply:
Ah, ok. It was slightly non-obvious why a popover library was needed at first